src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -52,7 +52,7 @@ public PersonCard(Person person, int displayedIndex) {
         phone.setText(person.getPhone().value);
         email.setText(person.getEmail().value);
 
-        if (person.getIsDone()) {
+        if (person.getIsDone().toString() == "True") {
```

Comment:
Strings are not primitive so they should be compared using the .equals method instead. 
[can see this link if u feel like it](https://stackoverflow.com/questions/7375427/comparing-two-identical-strings-with-returns-false)